.gitignore

```diff
@@ -1,5 +1,5 @@
 # Build results
-BuildOutput/
+/BuildOutput

```

Comment:
VS integration was showing the BuildOutput as a non-ignored folder. So I used VS to set the ignore and this is the syntax it uses. (It's a bit more correct as the intent is to ignore the %SPOCLIENT%\BuildOutput folder as opposed to any folder under %SPOCLIENT named BuildOutput)